daffodil-cli/src/main/scala/org/apache/daffodil/Main.scala

```diff
@@ -1440,11 +1440,10 @@ object Main {
               tunables, generateOpts.language)
 
             // Ask the CodeGenerator to generate source code from the DFDL schema
-            val rootNS = generateOpts.rootNS.toOption
             val outputDir = generateOpts.outdir.toOption.getOrElse(".")
             val rc = generator match {
               case Some(generator) => {
-                Timer.getResult("generating", generator.generateCode(rootNS, outputDir))
+                Timer.getResult("generating", generator.generateCode(outputDir))
```

Comment:
Is this just decluttering, or is there a different way to deal with namespaces here? Eventually we'll want to support them.